wp-includes/rest-api/endpoints/class-wp-rest-customize-changesets-controller.php

```diff
@@ -94,4 +133,835 @@ public function register_routes() {
 			'schema' => array( $this, 'get_public_item_schema' ),
 		) );
 	}
+
+	/**
+	 * Retrieves the changeset's schema, conforming to JSON Schema.
+	 *
+	 * @since 4.?.?
+	 * @access public
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/schema#',
+			'title'      => 'customize_changeset',
+			'type'       => 'object',
+			'properties' => array(
+				'author'          => array(
+					'description' => __( 'The ID for the author of the object.' ),
+					'type'        => 'integer',
+					'context'     => array( 'view', 'edit', 'embed' ),
+				),
+				'date'            => array(
+					'description' => __( "The date the object was published, in the site's timezone." ),
+					'type'        => 'string',
+					'format'      => 'date-time',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'arg_options' => array(
+						'sanitize_callback' => array( $this, 'sanitize_datetime' ),
+					),
+				),
+				'date_gmt'        => array(
+					'description' => __( 'The date the object was published, as GMT.' ),
+					'type'        => 'string',
+					'format'      => 'date-time',
+					'context'     => array( 'view', 'edit' ),
+					'arg_options' => array(
+						'sanitize_callback' => array( $this, 'sanitize_datetime' ),
+					),
+				),
+				'settings'        => array(
+					'description' => __( 'The content of the customize changeset. Changed settings in JSON format.' ),
+					'type'        => 'object',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'slug'            => array(
```

Comment:
I think that this should perhaps be `id` or `uuid` per the regex in the endpoint?